instaclone/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { CommentInput } from "./Styles";
+import Comment from "./Comment";
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: this.props.comments,
+      comment: "",
+      index: this.props.index
+    };
+  }
+  addNewComment = e => {
+    if (this.state.comment) {
+      let newComments = [
+        ...this.state.comments,
+        { username: localStorage.getItem("user"), text: this.state.comment }
+      ];
+      if (e.key === "Enter") {
+        this.setState({
+          comments: newComments,
+          comment: ""
+        });
+      }
+    }
+  };
+  deleteComment = index => {
```

Comment:
good job implementing this
my challenge to you is to change this methods functionality so only the comments that belong to the user can be deleted